static/app/components/events/breadcrumbs/breadcrumbsDrawer.tsx

```diff
@@ -207,8 +207,8 @@ export function BreadcrumbsDrawer({
   );
 
   return (
-    <BreadcrumbDrawerContainer>
-      <BreadcrumbDrawerHeader>
+    <EventDrawerContainer>
```

Comment:
changed the names of these components to be more generic so i could re-use them in the feature flag drawer